website/www/site/assets/js/language-switch-v2.js

```diff
@@ -25,6 +25,15 @@ function setScrollToNewPosition(clickedElementData) {
     $('html, body').scrollTop(elCurrentHeight - elPreviousOffsetFromViewPort);
 }
 
+function isFirefox() {
+    const userAgent = navigator.userAgent;
+    return userAgent.match(/firefox|fxios/i);
+}
```

Comment:
Functions that are now global in this file can be defined inside of `$(document).ready(function() {...` so as not to clutter the global namespace.